hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/listener/ClientSmartListenerService.java

```diff
@@ -192,7 +191,7 @@ public void run() {
                 for (Member member : memberList) {
                     try {
                         clientConnectionManager.getOrTriggerConnect(member.getAddress());
-                    } catch (IOException e) {
+                    } catch (HazelcastException e) {
```

Comment:
Should every HazelcastException be caught, or only the RetryableHazelcastExceptions?